server/views/subscription/web-subscribe.mjml.hbs

```diff
@@ -6,8 +6,5 @@
         <mj-text>
 {{> subscription_subscribe_form}}<!-- don't indent me! -->
         </mj-text>
-        <mj-button mj-class="button" href="#submit">
-            Subscribe to list
-        </mj-button>
```

Comment:
Uses the subscribe to list button in `server/views/subscription/partials/subscription-subscribe-form.hbs`